Tribler/Test/Core/Modules/RestApi/Channels/test_channels_torrents_endpoint.py

```diff
@@ -166,43 +167,39 @@ def test_add_torrent_from_url_to_channel_with_description(self):
         files_path = os.path.join(self.session_base_dir, 'http_torrent_files')
         os.mkdir(files_path)
         shutil.copyfile(TORRENT_FILE, os.path.join(files_path, 'ubuntu.torrent'))
+
         file_server_port = get_random_port()
         self.setUpFileServer(file_server_port, files_path)
 
         def verify_method_invocation(channel_id, torrent_def, extra_info=None, forward=True):
             self.assertEqual(my_channel_id, channel_id)
-            self.assertEqual(TorrentDef.load(os.path.join(files_path, 'ubuntu.torrent')), torrent_def)
+            self.assertEqual(TorrentDef.load(TORRENT_FILE), torrent_def)
             self.assertEqual({"description": "test"}, extra_info or {})
             self.assertEqual(True, forward)
 
         self.session.add_torrent_def_to_channel = verify_method_invocation
 
-        # Martijn: We are directly invoking the endpoint here. This is because the requests library used by
-        # TorrentDef.load_from_url is not working well when performing a request using do_request and seems to hang.
-        torrent_url = 'http://localhost:%s/ubuntu.torrent' % file_server_port
-        endpoint = ChannelModifyTorrentEndpoint(self.session, 'fakedispersyid', torrent_url)
-        fake_request = MockObject()
-        fake_request.content = MockObject()
-        fake_request.content.read = lambda: "description=test"
-        self.assertDictEqual(json.loads(endpoint.render_PUT(fake_request)), {"added": torrent_url})
+        torrent_url = 'http://localhost:%d/ubuntu.torrent' % file_server_port
+        url = 'channels/discovered/%s/torrents/%s' % ('fakedispersyid'.encode('hex'), urllib.quote_plus(torrent_url))
+        return self.do_request(url, expected_code=200, expected_json={"added": torrent_url}, request_type='PUT',
+                               post_data={"description": "test"})
 
     @deferred(timeout=10)
-    def test_add_torrent_from_magnet_to_channel_without_description(self):
+    def test_add_torrent_from_magnet_to_channel(self):
         """
         Testing whether adding a torrent with a magnet link to a channel without description works
         """
         my_channel_id = self.create_fake_channel("channel", "")
-        torrent_path = TORRENT_FILE
 
-        def fake_load_from_dht(_, callback):
-            meta_info = TorrentDef.load(torrent_path).get_metainfo()
+        def fake_get_metainfo(_, callback, timeout=10, timeout_callback=None, notify=True):
```

Comment:
You are not using most of these arguments, you can use `_dummy1`, `_dummy2` etc instead (so pylint won't give errors about unused arguments).